src/elements/GenresList.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Selection } from "@react-types/shared";
+import { IGenreData } from "../models/genreData";
+import { Listbox, ListboxItem } from "@nextui-org/react";
+
+function GenreList({ genres }: { genres: IGenreData[] }) {
+  //   state to track the selected genres
+  const [selectedKeys, setSelectedKeys] = React.useState<Selection>(
+    new Set([""])
+  );
+
+  // join the selected genres to get a search query
+  const selectedValue = React.useMemo(
+    () => Array.from(selectedKeys).join(", "),
+    [selectedKeys]
+  );
+
+  return (
+    <>
+      <h1>Explore genres</h1>
+      <div className="flex flex-col gap-2">
+        <div className="w-full border-small px-1 py-2 rounded-small border-default-200 dark:border-default-100">
```

Comment:
1. Is `border-small` a valid Tailwind class?

2. If you are looking to have a slighty rounded corner, I believe `rounded-sm` is the way to go.

Reply:
Yes, it is a next ui tailwind class 

```
.border-small {
    border-width: var(--nextui-border-width-small);
}
```